.github/workflows/manifest.yml

```diff
@@ -1,14 +1,12 @@
 name: Manifest
-
-on:
-  pull_request_target:
+on: pull_request_target
```

Comment:
I think this change might cause security issues.
pull_request_target actions are executed even on PR opened from non-contributors.

Is it possible to add malicious scripts in PR and execute it while running `west init -l .`?

Reply:
The removal of the `:` was just fixing a  mistake. This action does not execute any code at all from the codebase, it uses the action's code and the pre-installed west.

Reply:
Sorry, I was sure action type changes from pull_request to pull_request_target :/ 
You are right, I cannot find a way to run malicious code and PR can be merged.
However please remember that even small change in this workflow file or action can be problematic